pkgs/applications/audio/rofi-mpd/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, python37Packages, fetchFromGitHub }:
+
+with python37Packages; 
+buildPythonPackage rec {
+  pname = "rofi-mpd";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "JakeStanger";
+    repo = "Rofi_MPD";
+    rev = "168f3e7205761c960abd9b44c83f109cefcb9c2c";
+    sha256 = "0pdra1idgas3yl9z9v7b002igwg2c1mv0yw2ffb8rsbx88x4gbai";
+  };
+
+  propagatedBuildInputs = [ mutagen mpd2 ];
+
+  meta = with lib; {
+    description = "A rofi menu for interacting with MPD written in Python";
+    homepage = "https://github.com/JakeStanger/Rofi_MPD";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jakestanger ];
+    platforms = platforms.all;
+  };
+
+  doCheck = false;
```

Comment:
generally meta is the last thing in an expression, i would move doCheck between propagatedBuildInputs and meta